src/puppetlabs/puppetdb/cli/export.clj

```diff
@@ -219,7 +198,9 @@
   [args]
   (let [specs    [["-o" "--outfile OUTFILE" "Path to backup file (required)"]
                   ["-H" "--host HOST" "Hostname of PuppetDB server" :default "localhost"]
-                  ["-p" "--port PORT" "Port to connect to PuppetDB server (HTTP protocol only)" :parse-fn #(Integer. %) :default 8080]]
+                  ["-p" "--port PORT" "Port to connect to PuppetDB server (HTTP protocol only)" :parse-fn #(Integer. %) :default 8080]
+                  ["" "--url-prefix PREFIX" "Server prefix (HTTP protocol only)"
```

Comment:
So I know in config files in PE when we got to the point of having to specify all these things, we decided to just use a single `base-url` item that the user specifies (e.g. `http://localhost:8080/my_prefix`) instead, parsing it into the map when we received it. Might make things cleaner!